notebooks/official/tabular_workflows/tabnet_on_vertex_pipelines.ipynb

```diff
@@ -0,0 +1,1124 @@
+{
```

Comment:
<p>The link provides the official source for the dataset, but, in practice, we use a copy of the dataset hosted on Cloud Storage (?) gs://cloud-samples-data/vertex-ai/tabular-workflows/datasets/safe-driver/train.csv. Should we mention that here? Should we provide instructions for viewing the Cloud Storage dataset?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/726/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Makes sense. I'll add a sentence mentioning that and include a link to down the dataset in GCS directly.